user_docs/en/changes.t2t

```diff
@@ -10,6 +10,7 @@ What's New in NVDA
 - Added support for Bluetooth Low Energy HID Braille displays. (#15470)
 - The Add-on Store now supports installing add-ons in bulk by selecting multiple add-ons. (#15350)
 - From the add-on store, it's possible to open a dedicated webpage to see or provide feedback about the selected add-on. (#15576, @nvdaes)
+- A new "speech on demand" mode has been added. When speech is on demand, NVDA does not speak automatically (e.g. when moving the cursor) but still speaks when calling commands whose goal is explicitly to report something (e.g. report window title). (#481, @CyrilleB79)

```

Comment:
```suggestion
- A new "on-demand" speech mode has been added. When speech is on-demand, NVDA does not speak automatically (e.g. when moving the cursor) but still speaks when calling commands whose goal is explicitly to report something (e.g. report window title). (#481, @CyrilleB79)
```

Reply:
Fixed comments of @seanbudd and @XLTechie  on this line in 65dbc8ac6df72b1b73bcecb131d1fc20a96f127d.